web/src/pages/profile.tsx

```diff
@@ -275,6 +275,14 @@ const ProfilePage = (props: Props): ReactElement => {
     );
   };
 
+  const lockMunicipality = (): boolean => {
+    return (
+      !!profileData.municipality &&
+      (profileData.operatingPhase === "UP_AND_RUNNING_OWNING" ||
+        profileData.operatingPhase === "UP_AND_RUNNING")
```

Comment:
shouldn't this be based on whether or not tax access was success or not? This is just saying the field is locked for all Oscars and all up-and-running Poppys, always?